tests/jeos/glibc_locale.pm

```diff
@@ -38,6 +38,11 @@ my %test_data_lang = (
     de_DE => 'Eine Anleitung zum Melden von Programmfehlern finden Sie hier:'
 );
 
+sub switch_user {
+    enter_cmd("su - $testapi::username");
```

Comment:
I would add a comment explaining why this is better, so we will remember in 10 years, why this was done.

Reply:
Will this still run in 10 years? Who knows! But at least we could open this up in a nice pub and laugh at our past selves!